index.js

```diff
@@ -0,0 +1,59 @@
+const titleInputEl = document.getElementById('title-input');
+const authorInputEl = document.getElementById('author-input');
+const formEl = document.getElementById('form');
+const collectionsSectionEl = document.getElementById('books-collection');
+
+const collectionData = JSON.parse(window.localStorage.getItem('collectionArray')) || [];
+const removeButtons = [];
+
+const renderBooks = (collectionData) => {
+  collectionsSectionEl.innerHTML = '';
+  collectionData.forEach((book, index) => {
+    const bookEntry = document.createElement('div');
+    bookEntry.classList.add('book-entry');
+    const bookTitle = document.createElement('p');
+    bookTitle.classList.add('book-title');
+    bookTitle.textContent = book.title;
+    const bookAuthor = document.createElement('p');
+    bookAuthor.classList.add('book-author');
+    bookAuthor.textContent = book.author;
+    removeButtons[index] = document.createElement('button');
+    removeButtons[index].classList.add('remove-button');
+    removeButtons[index].textContent = 'Remove';
+    removeButtons[index].dataset.id = book.bookID;
+    bookEntry.append(bookTitle, bookAuthor, removeButtons[index]);
+    collectionsSectionEl.appendChild(bookEntry);
+  });
+
+  const addBook = (book) => {
+    collectionData.push(book);
+    window.localStorage.setItem('collectionArray', JSON.stringify(collectionData));
+  };
+
+  formEl.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const title = titleInputEl.value;
+    const author = authorInputEl.value;
+    const removeIndex = collectionData.length;
```

Comment:
- [ ] Kindly make sure to provide the unique `id`. At this moment we are having a conflict with ids please make sure to provide unique identification.

  _**HInt: `const removeIndex = Date.now()`**_

Reply:
Can you please provide more context on this review? we are struggling to understand why and how to refactor our code with respect to your suggestion. The bookID's are all unique when we check them on our end.

Reply:
@levy002